_PackagesTest/4ti2interface.html

```diff
@@ -0,0 +1,72 @@
+---
+name: 4ti2Interface
+dirname: 4ti2Interface-2019.09.02
+version: "2019.09.02"
+license: "GPL-2.0-or-later"
+date: 2019-09-02
+description: |
+    A link to 4ti2
```

Comment:
On the current website it says "interface" not "link"

Reply:
I compared with the old site. For me it says "link" in both versions.